src/containers/EventCreationPage.tsx

```diff
@@ -0,0 +1,163 @@
+    setEventName(e.target.value);
+  };
+  const handleDateChange = (e: any) => {
+    setEventDate(e.target.value);
+  };
+  const handleLocationChange = (e: any) => {
+    setEventLocation(e.target.value);
+  };
+
+  const handleNext = () => {
+    setActiveStep((prevStep) => prevStep + 1);
+  };
+  const handleBack = () => {
+    setActiveStep((prevStep) => prevStep - 1);
+  };
+
+  useEventMutation(
+    {
+      name: eventName,
+      eventDate: eventDate || new Date(),
+      createdBy: 1,
+      isActive: true,
+    },
+    complete,
+    setComplete
+  );
+
+  const handleComplete = () => {
+    setComplete(true);
+    window.location.href = "/events/";
+  };
+
+  const dateParts: {
+    year?: string;
+    month?: string;
+    day?: string;
+    literal?: string;
+  } = eventDate
+    ? new Intl.DateTimeFormat().formatToParts(eventDate).reduce(
+        (obj, currentPart) => ({
+          ...obj,
+          [currentPart.type]: currentPart.value,
+        }),
+        {}
+      )
+    : {};
+  const content =
+    activeStep === 0 ? (
+      <form>
+        <FormField
+          label="Event Name:"
+          placeholder="Event Name Here"
+          onChange={handleNameChange}
+          value={eventName}
+        />
+        <FormField
+          label="Date of Event:"
+          placeholder="YYYY:MM:DD"
+          onChange={handleDateChange}
+          value={
+            eventDate
+              ? `${dateParts.year}:${dateParts.month}:${dateParts.day}`
+              : ""
+          }
+          handleClick={handleOpenDateModal}
+        />
+      </form>
+    ) : (
+      <form>
+        <FormField
+          label="Event Location:"
+          placeholder="Enter Location Here"
+          onChange={handleLocationChange}
+          value={eventLocation}
+        />
+        <Map />
+      </form>
+    );
+
+  return (
+    <div className="landing-wrapper">
+      <div className="event-creation-top-section">
+        <div className="landing-top-bar">
+          <Typography variant="h3">Create New Event</Typography>
+          <div className="user-icon">
+            <Button
+              variant="outlined"
+              color="primary"
+              onClick={handleOpenCancelModal}
+            >
+              Cancel
+            </Button>
+          </div>
+        </div>
+      </div>
+      <div className="event-form">{content}</div>
+      <CancelModal
+        open={openCancelModal}
+        handleClose={handleCloseCancelModal}
+      />
+      <SelectDateModal
+        open={openDateModal}
+        handleClose={handleCloseDateModal}
+        eventDate={eventDate}
+        setEventDate={setEventDate}
+      />
+      <Stepper
+        activeStep={activeStep}
+        nextButton={
+          <div className="next-container">
+            <NextButton
+              handleClick={activeStep < 1 ? handleNext : handleComplete}
+              disabled={eventName === "" || eventDate === null}
+            />
```

Comment:
When `activeStep < 1` is `false`, you're passing the `handleComplete` which creates the event. The NextButton text should say "CREATE" (not "next"). I suggest you pass the text as a prop instead of hardcoding "NEXT".

Reply:
In the updated mockups, the next button text never changes to "CREATE" on the last step. I'm actually wondering if this was purposeful or a small error. I'll ask Gracie right now and update it when she confirms.
![image](https://user-images.githubusercontent.com/38408276/84959027-8f2b0680-b0cc-11ea-8285-c845b0e89f54.png)